### ISSUE TITLE
Noto Sans Mono's alphabet O and number 0 issue

### ISSUE THREAD

**Issue by YunsuChoi** [googlei18n/noto-fonts#887]
<img width="167" alt="2017-04-04 19 22 01" src="https://cloud.githubusercontent.com/assets/1073023/24652228/01180ba4-196c-11e7-843b-ed63e73fa997.png">

As we know, sometimes Alphabet Upper O or Lower case o is confused with number zero(0). especially for programming, it is very import issue. to solve this problem most of coding fonts(example D2 Coding) are has slash line in number zero. 
but as we see the attachment, Noto Sans Mono font is super confuse to identify O or 0. 
should fix this problem. for current state, Noto Sans Mono is unusable.

**Comment by daylightbrightledlight** [comment]
I agree with @dougfelt. The zero character should be slashed to distinguish it from the letter O character, especially in Noto Mono. I think @marekjez86 will help too.

This issue is related to the following: #561 #733 #755.

Please add the Priority-High label to this issue as well.

**Comment by danielb2** [comment]
A slash isn't the only option. Take 0 from Ubuntu's monospace 
<img width="30" alt="Screen Shot 2019-07-29 at 10 50 02" src="https://user-images.githubusercontent.com/26678/62070104-afb94c00-b1ee-11e9-8980-342abd702144.png">
it's a clean good look

**Comment by nizarsq** [comment]
@marekjez86 This issue related to https://github.com/googlefonts/noto-fonts/issues/733.
#733 resolved by using font-variant-numeric: slashed-zero (for CSS and HTML users).  Now, for none CSS and HTML users will 0030+FE00 be the only way to get slashed zero?
